backend/inspirehep/mailing/api/jobs.py

```diff
@@ -38,3 +42,64 @@ def send_jobs_weekly_campaign(html_content, test_emails=None):
 def subscribe_to_jobs_weekly_list(email, first_name, last_name):
     list_id = current_app.config["MAILCHIMP_JOBS_WEEKLY_LIST_ID"]
     return mailchimp_subscribe_user_to_list(list_id, email, first_name, last_name)
+
+
+def get_jobs_deadline_reminder_html_content(job):
+    host = current_app.config['SERVER_NAME']
+    return render_template("mailing/jobs/deadline_passed/base.html", job=job, host=host)
+
+
+def send_job_deadline_reminder(job):
+    """Send an email for the given expired job.
+
+    Args:
+        job (dict): a job record.
+
+    Return:
+        None
+    """
+    recipient = get_job_recipient(job)
+
+    if not recipient:
+        LOGGER.error(
+            'Cannot send deadline email: field acquisition_source.email not found',
+            recid=job['control_number']
+        )
+        return
+
+    sender = current_app.config["JOBS_DEADLINE_PASSED_SENDER_EMAIL"]
+    cc_addresses = [cd.get('email') for cd in job.get('contact_details')]
+    subject = f"Expired deadline for your INSPIRE job: {job['position']}"
+    content = get_jobs_deadline_reminder_html_content(job)
+
+    send_email(
+        sender=sender,
+        recipient=recipient,
+        cc=cc_addresses,
+        subject=subject,
+        content=content,
+    )
+
+
+def get_job_recipient(job):
+    """Get the best email to which send a job notification.
+
+    This function tries to load the latest email from the user who submitted
+    the job. If the user cannot be found, it returns the email from the
+    acquisition_source field.
+
+    Args:
+        job (dict): a job record.
+
+    Return:
+        string: the email for contacting the job author. It can be None.
+    """
+    recipient = None
+    user_id = job.get("acquisition_source", {}).get("internal_uid")
+    if user_id:
+        identity = UserIdentity.query.filter_by(id_user=user_id).one_or_none()
+        if identity:
+            recipient = identity.user.email
+    if not recipient:
+        recipient = job.get("acquisition_source", {}).get("email", "")
```

Comment:
```suggestion
        recipient = job.get_value("acquisition_source.email", "")
```

Reply:
that's not a dict but a `Hit` object. Apparently, using `get_value(job,  'acquisition_source.email')` to get nested fields here doesn't work